test/e2e/mcd_test.go

```diff
@@ -355,8 +355,16 @@ func TestReconcileAfterBadMC(t *testing.T) {
 	cs := framework.NewClientSet("")
 	bumpPoolMaxUnavailableTo(t, cs, 3)
 
-	// create a bad MC w/o a filesystem field which is going to fail reconciling
+	// create a MC that contains a valid ignition config but is not reconcilable
 	mcadd := createMCToAddFile("add-a-file", "/etc/mytestconfs", "test", "")
```

Comment:
you need to add `root` as the last arg to this func, right now it's creating an invalid ignition config anyway but we need a valid one with unreconcilable code (as you did below)

Reply:
thanks will. still unsure why i hit those weird e2e errors tho.